components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/proxy/NetworkingConfigurator.java

```diff
@@ -0,0 +1,57 @@
+package com.gradle.enterprise.proxy;
+
+import com.gradle.enterprise.cli.ConsoleLogger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.Properties;
+
+public class NetworkingConfigurator {
+
+    public static void configureNetworking(Optional<Path> networkSettingsFile, ConsoleLogger logger) {
+        networkSettingsFile.ifPresent(path -> {
+            try {
+                configureBasedOnProperties(path, logger);
+            } catch (IOException e) {
+                logger.debug("Unable to load settings from %s: %s", path, e.getMessage());
+                logger.debug(e);
+            }
```

Comment:
Should we throw here instead, given when this unlikely scenario happens, it is a serious issue that the user show know about?

Reply:
Makes sense to me, ✅ Done.